### DISCUSSION TITLE
Values at the cursor

### DISCUSSION THREAD

**Discussion by Breastfed** [collin80/SavvyCAN#289] [Ideas]
Hi Collin
I know I threw this out there before in another post ... 

Values at the cursor is an excellent feature, its difficult to view values across multiple traces zooming in and out looking at a specific point in the Time line.
As the mouse moves left and right the parameter Values in the top right corner are also updated dynamically.

I'd love to see this feature implemented. 

![image](https://user-images.githubusercontent.com/79179843/108791237-2e37c480-75b1-11eb-85c8-61415f635e7d.png)

Breastfed

**Comment by collin80** [comment]
I fully agree. It is a good idea and worthwhile to do. I don't know when it'll happen but it should happen. It's a good thing to implement. Perhaps I should take a break from coding to see if I can perfect cloning me three or four times. Though, my wife might say one of me is enough.
